### ISSUE TITLE
Name suggestions

### ISSUE THREAD

**Issue by alfredxing** [alfredxing/brick#12]
As @PascalPrecht mentioned in #9, Mozilla already has a popular project called Brick, so I'm taking feedback on whether to rename the project.

Comment with your thoughts on the issue. I'm also taking name suggestions if you want a rename.

Here are the criteria for new names:
1. Must be one word
2. Must have an available and reputable TLD (no `.io` please, they're really expensive. If you want to suggest `.io`, you can contribute the domain :smile:)

Thanks!

**Comment by alfredxing** [comment]
@fly I'd rather have a TLD that's at least partly recognized as a gTLD. Even better if it's on [this list](https://support.google.com/webmasters/answer/1347922).

**Comment by azer** [comment]
I also work on a project named Brick: http://github.com/azer/brick (dev branch will show my last commit)

**Comment by noahbuscher** [comment]
What about gypsum? It's used as a building material to hide up the complex mess of wood planks and wires in a building, much like how Brick obfuscates the messy task of downloading fonts and writing the appropriate CSS for them. Here some domains that you can currently get:
- gypsum.me
- gypsum.cc
- gypsum.ws
- gypsum.im
- gypsum.so

**Comment by alfredxing** [comment]
@Codingbean I'll consider it, but I don't particularly like the sound of the word...

**Comment by noahbuscher** [comment]
Yeah. Just an idea. I'm not the most creative person around. :wink:

**Comment by waldyrious** [comment]
How about Woffer? If you're willing to go for ccTLDs, there's `woff.er`, ~~or even wof.fr? :P~~ (update: scratch the latter, it's parked :/)

**Comment by alfredxing** [comment]
@waldir `.er` won't work, since the TLD is not publicly available (or if it is, it's extremely hard to get one).

**Comment by waldyrious** [comment]
I got an `.ir` (for wald.ir) a while ago through an Iranian friend, otherwise it would have cost me a lot more (and in any case I think they restrict it to Iranian citizens or companies). I don't know if such a strategy could help in this case, but I thought I'd mention it. But of course that depends on whether you like the name in the first place :)

**Comment by alfredxing** [comment]
It's an okay name, though I'd like to have something a bit more original (not as obvious).

**Comment by waldyrious** [comment]
Yeah, I wasn't terribly excited about it either, but I do think it's better than brick (I mean, does that even have a meaning? Perhaps it's too non-obvious to me) and if somehow we could get hold of an .er domain, that'd be sweet :)

**Comment by noahbuscher** [comment]
Just another idea: Slate. Think of a website as a blank **slate** that your site adds character to. There's most of the domains taken for this, but here's what's available:
- slate.ws
- slate.so
- slate.bz
- And of course: slate.sexy

**Comment by alfredxing** [comment]
@Codingbean I like that name, but the TLDs aren't as appealing.

**Comment by noahbuscher** [comment]
Agreed. You're kind of hard pressed to find a good one-word domain without making something up, or doing something like [Meeet](http://meeet.co) (adding an extra letter) or [Compilr](https://compilr.com/) (changing the spelling).

**Comment by alfredxing** [comment]
It doesn't have to be a dictionary word; most of those are already taken.

**Comment by WilliamOckham** [comment]
I suggest Featurizer (font.featurizer.com seems like a great url for this project) and it appears to be available on all the tlds.

**Comment by muescha** [comment]
I would open it to a 2 words name and include the main target "font" to the name

**Comment by muescha** [comment]
btw: brick was a horrible name. If you search for it you never got this repo at google: font brick

**Comment by alfredxing** [comment]
@muescha I'm mainly targeting the keyword "webfont" here. Anyway, I like doing SEO by popularity, not by injecting keywords at every possibility.

I just checked and we're on the first page of [Google results for the search "webfonts"](http://lmgtfy.com/?q=webfonts), which is pretty good already. And it'll probably rise once this project ages a bit.

**Comment by curiositry** [comment]
**Ligature.co?** 

It’s available — and even on sale with Godaddy, if you want to go with them for some strange reason.

**Comment by bsdlp** [comment]
Ligature.co is a great idea IMO, but stay away from godaddy.

Would recommend:
- internet.bs
- gandi.net
- name.com
- namecheap.com

In no particular order

**Comment by alfredxing** [comment]
@fly Considering it, but it wouldn't really work SEO-wise, even with the phrase "Ligature webfonts"...

**Comment by KyleAMathews** [comment]
FWIW, I think brick is fine name. Perhaps not ideal but since it's not objectionable and already has a decent bit of mindshare/googleshare, I don't see any reason to change it.

**Comment by alfredxing** [comment]
@KyleAMathews Thanks! That's what's I'm thinking too, especially after this long...

Still open to suggestions though!

**Comment by jakeburden** [comment]
Silica.io [is available](https://domai.nr/silica/with/silica.io), silica [doesn't seem to be popular project name on github](https://github.com/search?p=2&q=silica&type=Repositories), and silica is [what bricks are mostly made of](https://en.wikipedia.org/wiki/Brick#Methods_of_manufacture).

I also think Brick is a fine name.

**Comment by alfredxing** [comment]
@JacobJWBurden Seems to be a bit unrelated, though, despite it being the compound that makes up bricks. "Brick" is more of a generic term, and here I intend it to mean strong, sturdy, classic, simple, and a part [of something bigger].

**Comment by alfredxing** [comment]
Any other suggestions before I close this issue? I had originally wanted to get some money to purchase `brick.io`, but it seems that's taken. I still want that gTLD so (non-)targeting is a bit easier.
